Aliases/kubeprod@1.2.rb

```diff
@@ -0,0 +1 @@
+../Formula/kubeprod.rb
```

Comment:
Alias needs to be removed, as versioned formula was declined.

Reply:
done!